tmca-tester/tmca-tester.js

```diff
@@ -229,26 +229,59 @@ function testSignature(signedMessage)
   request.end();
 }
 
-menu.disableDefaultHeader()
-  .addDelimiter('-', 40, 'TMC REST Test Menu')
-  .addItem(
-    'Test sign map message',
-    testSignMap
-  )
-  .addItem(
-    'Test sign tim message',
-    testSignTim
-  )
-  .addItem(
-    'Test sign tim message with validity override (example: 3 3600)',
-    function (overrride) {
-      testSignTimValidityOverride(overrride);
-    },
-    null,
-    [
-      {'name': 'override', 'type': 'numeric'}
-    ]
-  )
-  .addDelimiter('*', 40)
-  .start()
+function mainMenu() {
+  console.log('TMC REST Test Menu');
+  console.log('-'.repeat(40));
+  console.log('1. Test sign map message');
+  console.log('2. Test sign tim message');
+  console.log('3. Test sign tim message with validity override');
+  console.log('4. Exit');
+  
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
+
+  rl.question('Select an option (1-4): ', async (answer) => {
+    switch (answer) {
+      case '1':
+        await new Promise((resolve) => {
```

Comment:
Note: Currently, these options use a fixed timeout to avoid printing output to the console before the menu is re-displayed. If we want to avoid all instances of unexpected console output, it would be worth our time to convert the `testSignMap`, `testSignTim`, and `testSignTimValidityOverride` functions to be asynchronous. However, I wasn't sure if this was in the scope of this PR.